text/0010-keptn-cli-enhancements.md

```diff
@@ -0,0 +1,62 @@
+# keptn cli and API enhancements
+
+Enhance keptn-cli with the ability to list and get details of onboarded projects and services.
+
+## Motivation
+
+Right now when I'm using keptn within a CI/CD Pipeline I want to check if my service is already onboarded in keptn, if not it should be done automatically.
+From Ops-Perspective I want to get a list of all projects which are using keptn. Like in helm a keptn list command to retrieve a list of all projects and for more details a keptn get/details <project> command which returns all services within the project would be benificial.
+As workaround I'm checking if in my kubernetes cluster a lighthouse-configmap is existing, but this is not a pretty solution.
+
+## Explanation
+
+```bash
+# keptn list
```

Comment:
I'd like to be a little bit more generic here, e.g.:
```console
keptn get projects
```

Reply:
Like with `kubectl` no differentiation between singular and plural should be made. 
For example, `keptn get projects` returns the same like `keptn get project`